src/main/java/org/somerville/swag/data/entity/Order.java

```diff
@@ -40,4 +40,9 @@ public void setCustomerId(int customerId) {
     public void setOrderLinesList(ArrayList<OrderLine> orderLinesList) {
         this.orderLinesList = orderLinesList;
     }
+
+    public ArrayList<OrderLine> addOrderLineToOrder(ArrayList<OrderLine> existingOrderLines, OrderLine orderLine){
```

Comment:
- Can reduce the complexity of this method by removing existingOrderLines parameter as you are already within Order.java. 
- Method name could also be named to "addOrderLine" for increased readability.

Good job overall!